### ISSUE TITLE
Stuck Job with Worker Group Concurrency + Job Group / Priority 

### ISSUE THREAD

**Issue by lukepolo** [taskforcesh/bullmq-pro-support#76]
While testing group priroirty feature we ran into a case where a job would get stuck and would not get processed. 

![image](https://github.com/taskforcesh/bullmq-pro-support/assets/2066668/7527ff1d-4ec7-4e44-a2da-25569a9e6674)

![image](https://github.com/taskforcesh/bullmq-pro-support/assets/2066668/766556f5-4800-499d-a421-1fcb93878ebe)

```
{
  "jobId": "fc98faa9efd9ead25df539ed90280aadc6c17901e9fc006e98938e002310a60c-UpdateGatewayVersion",
  "removeOnFail": true,
  "backoff": {
    "delay": 1000,
    "type": "exponential"
  },
  "group": {
    "id": "10.43.10.10",
    "priority": 1
  },
  "removeOnComplete": true,
  "delay": 0,
  "attempts": 3
}
```

working on figuring out why at the moment. Ive even tried removing the job before adding it , which says it was removed, but it seems it did not

**Comment by lukepolo** [comment]
ok it semes to be there can be an issue that that i remove a job but it doesn't remove it from the priority list , and then tries to process the job, but its missing the details of that job. 

So there may be an issue where removing a job doens't remove it from the priority list (note this job that im removing was in a delayed state , then added 

![image](https://github.com/taskforcesh/bullmq-pro-support/assets/2066668/e62f198a-2187-463a-87d1-f8885e15d3bc)



notice the job details isnow missing , but its still in the priority listing

**Comment by roggervalf** [comment]
thank you @lukepolo I opened a fix pr for this case

**Comment by roggervalf** [comment]
fix is available in v7.8.3 @lukepolo

**Comment by lukepolo** [comment]
awesome thanks!!!
